src/js/component/recorder.vue

```diff
@@ -3,35 +3,29 @@
         v-bind:class="{
             'recorder--recording': isRecording
         }"
-        v-if="isSupported">
+        >
         <button
             class="recorder__record-btn"
-            @click="startRecording">
-            <div class="recorder__record-btn__content" v-if="!isRecording">
-                <span class="material-icons recorder__record-btn__content__icon">search</span>
-                Look for a movie
-            </div>
-            <div class="recorder__record-btn__content" v-if="isRecording">
+            @click="$emit('startRecording')">
```

Comment:
Use parent-child communication using callback.
I find it easier to work with. Simpler.